tests/test_amber_serializer.py

```diff
@@ -51,6 +51,7 @@ def test_dict(snapshot, actual):
 
 def test_set(snapshot):
     assert snapshot == {"this", "is", "a", "set"}
+    assert snapshot == {"contains", "frozen", frozenset({"1", "2"})}
```

Comment:
Nice!

Reply:
Also file can be renamed back to `tests/test_snapshots.py`

Reply:
actually renaming the "plugin" tests to `test_image_serializer` format